.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -192,6 +192,9 @@ jobs:
           docker run --rm -v "$(pwd)/../":/v -w /v "${ALPINE_IMAGE}" chown -R "$(id -u):$(id -g)" .
       - name: Checkout PyTorch
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # deep clone, to allow sharding to use git rev-list
```

Comment:
I think fetch-depth: 0 in this scenario adds ~2min to the checkout process here since we have a lot of branches to fetch, perhaps we can do the submodule checkout without adding this?